File: auth/src/main/java/com/harrish/auth/dto/BlogPostResponse.java
package com.harrish.auth.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Getter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class BlogPostResponse {
    private Long id;
    private String title;
    private String content;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
    private UserDto createdBy;
    private UserDto updatedBy;
}

File: auth-kotlin/src/main/kotlin/com/harrish/auth/dto/BlogPostResponse.kt
package com.harrish.auth.dto

import java.time.LocalDateTime

data class BlogPostResponse(
    val id: Long,
    val title: String,
    val content: String,
    val createdAt: LocalDateTime?,
    val updatedAt: LocalDateTime?,
    val createdBy: UserDto?,
    val updatedBy: UserDto?
)
